### PR TITLE
Fix #396 -- Spells related

### LINKED ISSUE #396: Fix summoner spells cast time and instant cast flag checking
Title

---
Beep, boop, I'm [a bot](https://github.com/LeagueSandbox/IssueBot)! This issue was created by `@joaquin95` in `#game-server`.

### PR DESCRIPTION
**Pull request by ghost**
Summoner spells stats are correctly loaded now.
Instant cast flag is correctly checked now.

Refs #396

### CODE REVIEW

File: GameServerLib/Logic/GameObjects/Spell.cs [hunk 1]
```diff
@@ -126,22 +126,6 @@ public Spell(Champion owner, string spellName, byte slot)
 
             JObject data;
 
-            if (slot > 3)
```
Comment (molenzwiebel) [inline GameServerLib/Logic/GameObjects/Spell.cs:129]: Why was this removed? Was it simply obsolete?

Comment (ghost) [reply GameServerLib/Logic/GameObjects/Spell.cs:129]: The code that follows that block does the same and more. Returning at that point means that the cast time is never loaded from the files

Comment (molenzwiebel) [reply GameServerLib/Logic/GameObjects/Spell.cs:129]: Ah ok :+1:

**Review by molenzwiebel — APPROVED** [review]
